force_wfmanager/wfmanager_setup_task.py

```diff
@@ -666,3 +681,39 @@ def switch_task(self):
 
     def exit(self):
         self.window.close()
+
+    # Custom UI Methods
+    def ui_select(self):
```

Comment:
We could also move `selected_contributed_ui` to become an attribute of `SystemState` if we wanted to separate the custom UI from the setup pane. This isn't necessary at the moment, but could be done at a later stage depending on how we choose to deliver a 'streamline' version of the app.